topdown/time.go

```diff
@@ -186,9 +186,8 @@ func builtinDiff(a, b ast.Value) (ast.Value, error) {
 	}
 	// END REDISTRIBUTION FROM APACHE 2.0 LICENSED PROJECT
 
-	result := ast.NewArray(ast.IntNumberTerm(year), ast.IntNumberTerm(month), ast.IntNumberTerm(day),
-		ast.IntNumberTerm(hour), ast.IntNumberTerm(min), ast.IntNumberTerm(sec))
-	return result, nil
+	return iter(ast.NewTerm(ast.NewArray(ast.IntNumberTerm(year), ast.IntNumberTerm(month), ast.IntNumberTerm(day),
```

Comment:
[nit] Isn't that the same as
```suggestion
	return iter(ast.ArrayTerm(ast.IntNumberTerm(year), ast.IntNumberTerm(month), ast.IntNumberTerm(day),
```

Reply:
Indeed, thanks